zppy/e3sm_diags.py

```diff
@@ -222,7 +222,7 @@ def e3sm_diags(config, scriptDir, existing_bundles, job_ids_file):  # noqa: C901
                 p.pprint(c)
                 p.pprint(s)
 
-            export = "NONE"
+            export = "ALL"
```

Comment:
Set export to "ALL" to load all environment.

Reply:
Ah, that makes sense.